torchaudio/__init__.py

```diff
@@ -338,25 +339,65 @@ def get_sox_bool(i: int = 0) -> Any:
         return _torch_sox.sox_bool(i)
 
 
+_SOX_INITIALIZED = False
+# This variable has a micro lifecycle. (False -> True -> None)
+# False: Not initialized
+# True: Initialized
+# None: Already shut down (should not be initialized again.)
+
+_SOX_SUCCESS_CODE = 0
+# defined at
+# https://fossies.org/dox/sox-14.4.2/sox_8h.html#a8e07e80cebeff3339265d89c387cea93a9ef2b87ec303edfe40751d9a85fadeeb
```

Comment:
It kind of bugs me to have duplicated constant definition here.
An alternative approach would be, changing the CPP impl of `initialize_sox` to return boolean, indicating success / failure by performing return code check on CPP side.

Reply:
This can be done in a separate PR later.